src/helpers/time.ts

```diff
@@ -22,5 +22,14 @@ export function parseDurationString(durationString: string) {
     throw new Error("Invalid duration string format.");
   }
   const [, value] = match;
+  // Custom support for months, based on a 30 days duration
+  if (/months?/i.test(value)) {
+    const monthsMatch = value.match(/\d+/);
+    if (monthsMatch) {
+      return Duration.fromObject({ months: Number(monthsMatch[0]) }).shiftToAll();
```

Comment:
`Duration.fromObject` I am not familiar with this method, or object. I thought you said there was no support for months, so I assumed you would manually handle it as "30 days" at this line of code.

Reply:
Correct, `ms` cannot convert months to milliseconds, however `Luxon` considers a month as 30 days (`Duration` comes from it) which is why I manually extract the duration.